pysat/tests/test_registry.py

```diff
@@ -1,9 +1,12 @@
 """
 tests the registration of user-defined modules
 """
+
+import numpy as np
 import pytest
 import sys
 
+import pysat
 from pysat import user_modules
 from pysat.utils import registry
 from pysat import Instrument
```

Comment:
Since you're now importing pysat, is it really useful to separately importing `user_modules` and `Instrument`?